docs/how-to-build-with-ddn/with-stripe.mdx

```diff
@@ -0,0 +1,243 @@
+
+This tutorial assumes you're starting with an existing [Stripe account](https://stripe.com/) which has some
+[products](https://docs.stripe.com/api/products) and [checkout sessions](https://docs.stripe.com/api/checkout/sessions).
+Though, by the end, you'll be able to query any information stored in Stripe.
+
+Hasura will never modify your source schema.
+
+<Prereqs />
+
+## Tutorial
+
+### Step 1. Authenticate your CLI
+
+```sh title="Before you can create a new Hasura DDN project, you need to authenticate your CLI:"
+ddn auth login
+```
+
+This will launch a browser window prompting you to log in or sign up for Hasura DDN. After you log in, the CLI will
+acknowledge your login, giving you access to Hasura Cloud resources.
+
+### Step 2. Scaffold out a new local project
+
+```sh title="Next, create a new local project:"
+ddn supergraph init my-project && cd my-project
+```
+
+Once you move into this directory, you'll see your project scaffolded out for you. You can view the structure by either
+running `ls` in your terminal, or by opening the directory in your preferred editor.
+
+### Step 3. Initialize your Stripe connector
+
+```sh title="In your project directory, run:"
+ddn connector init my_stripe -i
+```
+
+From the dropdown, select `hasura/stripe` (you can type to filter the list), then hit enter to accept the default of all
+the options.
+
+You'll be prompted for an environment variable:
+
+| Variable                   | Description                                                                                                    | Example                               |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------- | ------------------------------------- |
+| `STRIPE_BEARER_AUTH_TOKEN` | An auth token retrievable from your [API Keys dashboard in Stripe](https://dashboard.stripe.com/test/apikeys). | `sk_<test\|prod>_<unique_identifier>` |
+
+### Step 4. Introspect your Stripe account
+
+```sh title="Next, use the CLI to introspect your Stripe account:"
+ddn connector introspect my_stripe
+```
+
+After running this, you should see a representation of the Stripe API's schema in the `app/metadata/my_stripe.hml` file;
+you can view this using `cat` or open the file in your editor.
+
+Under the hood, the Stripe connector utilizes the [`ndc-http` connector](https://github.com/hasura/ndc-http) via
+[Stripe's OpenAPI spec](https://github.com/stripe/openapi). This ensures you can incorporate any resources available via
+Stripe's REST API into your supergraph.
+
+```sh title="Additionally, you can check which resources are available — and their status — at any point using the CLI:"
+ddn connector show-resources my_stripe
+```
+
+### Step 5. Add your command
+
+```sh title="Let's start by tracking the GetProducts endpoint from Stripe's API:"
+ddn command add my_stripe "GetProducts"
```

Comment:
I don't mind the quotes but we should be consistent throughout docs. It works without quotes too.